sdk/go/aws/alb/loadBalancer.go

```diff
@@ -202,9 +202,9 @@ type loadBalancerArgs struct {
 	// If true, the LB will be internal.
 	Internal *bool `pulumi:"internal"`
 	// The type of IP addresses used by the subnets for your load balancer. The possible values are `ipv4` and `dualstack`
-	IpAddressType *string `pulumi:"ipAddressType"`
+	IpAddressType interface{} `pulumi:"ipAddressType"`
 	// The type of load balancer to create. Possible values are `application` or `network`. The default value is `application`.
-	LoadBalancerType *string `pulumi:"loadBalancerType"`
+	LoadBalancerType interface{} `pulumi:"loadBalancerType"`
```

Comment:
Is this expected?  Is it a breaking change?  Will we no longer be able to safely add AltTypes once Go is GA?

cc @pgavlin @EvanBoyle

Reply:
> Is this expected? Is it a breaking change? 

This is expected. It is not a breaking change, as it only allows additional types for this property.

> Will we no longer be able to safely add AltTypes once Go is GA?

Not without a breaking change, no. However, fixing this s.t. the primitive type is used for AltTyped (`oneOf` in the Pulumi schema) properties would allow us to add proper AltTypes in Go w/o a breaking change.